### PR TITLE
Added VHS syntax package

### PR DESCRIPTION
**Pull request by patopesto**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is a syntax highlighter for charmbracelet's [VHS](https://github.com/charmbracelet/vhs) tape files

There are no packages like it in Package Control.

### CODE REVIEW

**Comment by patopesto** [comment]
The repository actually contains a top level LICENSE file. Not entirely sure how to handle the bot's warning

**Comment by braver** [comment]
That's a bit of a weird test result. Maybe the script just got confused, because your repo does seem to be missing a syntax file? 🧐

**Comment by patopesto** [comment]
Oops terribly sorry. I've now committed the forgotten syntax file.

**Comment by braver** [comment]
Not sure what the bot is on about… anyway

- Your variable.annotation scope is missing the tape suffix
- Perhaps you could specify those keywords a bit?
- In your PR here you don’t have to specify the author, it will pick up your GitHub username automatically

**Comment by patopesto** [comment]
Thanks for the feedback. 
I updated the missing suffix and the tag.
However I'm not sure I totally understand what you mean by "specifying the keywords a bit". The language is very simple and only contains a specific set of commands/keywords which are parsed. Are you suggesting parsing each individual commands?

**Comment by braver** [comment]
> The language is very simple and only contains a specific set of commands/keywords which are parsed

It seemed like you have two sets of keywords, keys and instructions and they might map to one of the [common groups of keywords](https://www.sublimetext.com/docs/scope_naming.html#keyword).. or not, also fine 🙂 

In any case something you can refine later.

File: repository/v.json [hunk 1]
```diff
@@ -317,6 +317,18 @@
 				}
 			]
 		},
+		{
+			"name": "VHS",
+			"details": "https://github.com/patopesto/Sublime-Text-VHS-Syntax",
+			"labels": ["language syntax"],
+			"author": "patopesto",
```
Comment (braver) [inline repository/v.json:324]: Since this is just your GitHub username you can omit it. 

```suggestion
```

**Review by braver — CHANGES_REQUESTED** [review]

**Review by braver — APPROVED** [review]

**Pull request merged**